pinot-segment-local/src/main/java/org/apache/pinot/segment/local/realtime/impl/json/MutableJsonIndexImpl.java

```diff
@@ -342,6 +424,11 @@ public String[] getValuesForKeyAndDocs(int[] docIds, Map<String, RoaringBitmap>
     return values;
   }
 
+  private Map<String, RoaringBitmap> getMatchingKeysMap(String key) {
+    return _postingListMap.subMap(key + BaseJsonIndexCreator.KEY_VALUE_SEPARATOR, false,
+        key + BaseJsonIndexCreator.KEY_VALUE_SEPARATOR_NEXT_CHAR, false);
```

Comment:
```suggestion
    return _postingListMap.subMap(key + JsonIndexCreator.KEY_VALUE_SEPARATOR, false,
        key + JsonIndexCreator.KEY_VALUE_SEPARATOR_NEXT_CHAR, false);
```

Reply:
Also modify `ImmutableJsonIndexReader` to use this constant

Reply:
Ack